src/media-groups.js

```diff
@@ -568,7 +568,7 @@ export const initialize = {
           const track = tech.addRemoteTextTrack({
             id: properties.instreamId,
             kind: 'captions',
-            enabled: false,
+            enabled: properties.default && properties.autoselect,
```

Comment:
the property name should be `default`

Reply:
properties.default as opposed to (properties.default && properties.autoselect)?

Reply:
instead of `enabled`, see https://github.com/videojs/http-streaming/pull/239/files#diff-46e43d417013a9cfb72f6b3210979b63R515